rosdep/base.yaml

```
@@ -3758,6 +3758,10 @@ libmlpack-dev:
   debian: [libmlpack-dev]
   fedora: [mlpack-devel]
   ubuntu: [libmlpack-dev]
+libmnl-dev:
+  debian: [libmnl-dev]
+  fedora: [libmnl-devel]
+  ubuntu: [libmnl-dev]
```

```suggestion
libmnl-dev:
  alpine: [libmnl-dev]
  debian: [libmnl-dev]
  fedora: [libmnl-devel]
  opensuse: [libmnl-devel]
  rhel: [libmnl-devel]
  ubuntu: [libmnl-dev]
```
Mind adding these to your rosdep key?

* https://pkgs.alpinelinux.org/package/edge/main/x86_64/libmnl-dev
* https://software.opensuse.org/package/libmnl
* http://mirror.centos.org/centos-8/8/AppStream/x86_64/os/Packages/libmnl-devel-1.0.4-6.el8.i686.rpm